src/man/sssd.conf.5.xml

```diff
@@ -988,6 +988,93 @@ fallback_homedir = /home/%u
                     </listitem>
                 </varlistentry>
                 <varlistentry>
+                    <term>memcache_size (integer)</term>
+                    <listitem>
+                        <para>
+                            Number of slots allocated inside fast in-memory
+                            caches. Note that one entry in fast in-memory
+                            cache can occupy more than one slot. Setting
+                            the size to 0 will disable the in-memory caches.
+                            This option can be overriden by
+                            memcache_size_passwd, memcache_size_group and
+                            memcache_size_initgroups options for passwd, group
+                            and initgroups in-memory caches respectively.
+                        </para>
+                        <para>
+                            Default: 50000
+                        </para>
+                        <para>
+                            NOTE: If the environment variable
+                            SSS_NSS_USE_MEMCACHE is set to "NO", client
+                            applications will not use the fast in-memory
+                            cache.
```

Comment:
@mzidek-rh, here (and in all other places in the docs) I'd keep the warning mentioning that we strongly recommend to not disable memcache due to the several performance issues that whoever does that will face.